### ISSUE TITLE
GML Conformance classes: Upload limit?

### ISSUE THREAD

**Issue by ghost** [INSPIRE-MIF/helpdesk-validator#322]
Hi all,

could someone please answer following questions:
1. What is the maximum size for GML-Upload in INSPIRE Validator **staging** instance?
2. What is the maximum size for GML-Upload in INSPIRE Validator **production** instance?

Thanks in advance.

**Comment by carlospzurita** [comment]
Dear @DeordD 
The size limit in production is 20MB. In staging, this limit is reduced to 2 MB. We are increasing the latter to the same as in production.

**Comment by carlospzurita** [comment]
The limit in staging is upped to 20MB

**Comment by ghost** [comment]
Hi @carlospzurita ,

thanks for quick response!

Maybe it would be good to provide correct values in the test configuration window:

- production 1 GB "allowed" (real value 20 MB)
![grafik](https://user-images.githubusercontent.com/38787131/83639498-0d56bb80-a5ab-11ea-8a3c-50bd40c2789c.png)

- staging 740 MB "allowed" (real value 20 MB)
![grafik](https://user-images.githubusercontent.com/38787131/83639796-7e966e80-a5ab-11ea-8dbc-7194fbc2b30e.png)

**Comment by carlospzurita** [comment]
Thank you for this @DeordD we will analyze the issue and get back to you with more feedback

**Comment by carlospzurita** [comment]
Dear @DeordD 

We have increased the upload size limit to 200MB on the [staging](http://staging-inspire-validator.eu-west-1.elasticbeanstalk.com/etf-webapp/) instance. After an internal discussion, we decided that a much higher limit was needed. We will keep this change for now on staging until the new release, to check if this causes any issue.

We have also updated the tooltip on the "Start Test Run" dialog.
